sdk/communication/Azure.Communication.CallingServer/tests/CallConnection/CallConnectionLiveTests.cs

```diff
@@ -175,60 +175,6 @@ public async Task RunCreateAddMuteGetParticipntRemoveHangupScenarioTests()
             }
         }
 
-        [Test]
-        public async Task RunCreateAddHoldResumeAudioRemoveHangupScenarioTests()
```

Comment:
Let us keep this test and rename Hold to RemoveFromDefaultAudioGroup and rename Resume to AddToDefaultAudioGroup